### PR TITLE
refactor:Use Civil Jurisdiction and Unspecified Claims case Type (CMC-462))

### PR DESCRIPTION
**Pull request by dharmendrak**
### JIRA link (if applicable) ###

https://tools.hmcts.net/jira/browse/CMC-462

### Change description ###

Use Civil Jurisdiction and Unspecified Claims case Type 

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

**Review by sabahirfan — DISMISSED** [review]

File: e2e/pages/createClaim/createCase.page.js [hunk 1]
```diff
@@ -10,8 +10,8 @@ module.exports = {
   startButton: 'Start',
 
   async selectCaseType() {
-      I.selectOption(this.fields.jurisdiction, 'Civil Money Claims');
-      I.selectOption(this.fields.caseType, 'Personal Injury');
+      I.selectOption(this.fields.jurisdiction, 'Civil Jurisdiction');
```
Comment (swalker125) [inline e2e/pages/createClaim/createCase.page.js]: I think this should just be Civil as I believe name shows up

```
  {
    "LiveFrom": "01/01/2017",
    "ID": "CIVIL",
    "Name": "Civil",
    "Description": "Civil Jurisdiction",
    "Shuttered": false
  }
  ```

File: src/main/java/uk/gov/hmcts/reform/ucmc/controllers/CallbackController.java [hunk 2]
```diff
@@ -48,7 +48,7 @@ public CallbackResponse callback(
         @NotNull @RequestBody CallbackRequest callback,
         @PathVariable("version") Optional<CallbackVersion> version
     ) {
-        log.info("Received callback from CCD, eventId: {}", callback.getEventId());
+        log.info("Received callback from CCD, eventId: {} and callback type: {}", callback.getEventId(), callbackType);
```
Comment (swalker125) [inline src/main/java/uk/gov/hmcts/reform/ucmc/controllers/CallbackController.java]: good change, but not relevant to this PR

**Review by swalker125 — DISMISSED** [review]
if we're happy to use conventional commits, this would be a refactor. not a feature I think 👍

**Review by swalker125 — COMMENTED** [review]
did you intened to add civil-unspecified-docker?

**Comment by sabahirfan** [comment]
Would you please change the roles in `SecurityConfiguration` class accordingly?

**Review by sabahirfan — DISMISSED** [review]
LGTM

**Review by ckalista — DISMISSED** [review]

**Review by ckalista — DISMISSED** [review]

**Review by sabahirfan — DISMISSED** [review]

**Review by sabahirfan — APPROVED** [review]

File: src/main/java/uk/gov/hmcts/reform/ucmc/controllers/CallbackController.java (hunk 2 above)
Comment (dharmendrak) [reply src/main/java/uk/gov/hmcts/reform/ucmc/controllers/CallbackController.java]: Wass removed

**Review by ckalista — APPROVED** [review]

**Pull request merged**